install-scripts/windows/wsl-install.sh

```diff
@@ -0,0 +1,50 @@
+#Updated Script file
+#!/bin/bash
+
+# Define variables
+repo_version=$(if command -v lsb_release &> /dev/null; then lsb_release -r -s; else grep -oP '(?<=^VERSION_ID=).+' /etc/os-release | tr -d '"'; fi)
+dotnet_pkg_url="https://packages.microsoft.com/config/ubuntu/$repo_version/packages-microsoft-prod.deb"
+splashkit_url="https://raw.githubusercontent.com/splashkit/skm/master/install-scripts/skm-install.sh"
+
+# Update package lists
+sudo apt-get update
+
+# Install required packages
+sudo apt-get -y install wget git curl
+
+# Install SplashKit
+echo "Installing SplashKit..."
+bash <(curl -s $splashkit_url)
+echo 'export PATH=$PATH:~/.splashkit' >> ~/.bashrc
+export PATH=$PATH:~/.splashkit
+yes | skm linux install
+if [ -d ~/.splashkit/global ]; then
```

Comment:
Just curious as to why the existence of this directory is checked? 
Is there a situation where the global directory doesn't exist after cloning skm?